knora/onto-visualize.py

```diff
@@ -0,0 +1,59 @@
+import pygraphviz as pgv
```

Comment:
Is it possible you added stuff that was _not_ within the scope of the PR? ;-)

Reply:
Vorläufig wegkopiert :-)